modules/serverless-install-eventing-yaml.adoc

```diff
@@ -19,6 +19,7 @@ ifdef::openshift-dedicated[]
 endif::[]
 
 * You have installed the {ServerlessOperatorName}.
+* You have installed the `oc` CLI.
```

Comment:
This prereq should be “Install the OpenShift CLI (`oc`)” per the [contrib guide](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#openshift-cli). I suggest editing the first one, too.

> Use list items that are grammatically parallel.

Reply:
I honestly had no idea! As soon as you said it I knew it was wrong in a bunch of other places too, so this PR may have gotten a lot bigger - I did a quick sweep of all the modules to update this before I forgot. Thanks for the heads up!